lib/rubocop/cop/factory_bot/association_style.rb

```diff
@@ -67,6 +67,7 @@ class AssociationStyle < ::RuboCop::Cop::Base # rubocop:disable Metrics/ClassLen
 
         DEFAULT_NON_IMPLICIT_ASSOCIATION_METHOD_NAMES = %w[
```

Comment:
I didn't change this, but I was wondering if this is the same as `DEFAULT_EXPLICIT_ASSOCIATION_METHOD_NAMES`. The naming/use of `non_implicit` vs. using `explicit` for the method names & constants was tripping me up. If `non_implicit` == `explicit` I can do a separate PR to change these

Reply:
There are three styles, explicit, implicit and [inline](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#inline-definition). So those non-implicit are explicit/inline.

Reply:
That makes sense to me now, thank you. I saw brief mention of the inline but it didn't click in my head